Add warning for call async function from a thread (#409)
home-assistant/developers.home-assistant: docs/asyncio_working_with_async.md

    return f"Hello {target}!"
```

> **Warning:** be carfuly with this! If the async function using executor jobs, you can lead a deadlock.

Reviewer: ```suggestion
**Warning:** be careful with this! If the async function uses executor jobs, it can lead to a deadlock.
```
